lib/angular-rails-templates/engine.rb

```diff
@@ -10,7 +10,7 @@ class Engine < ::Rails::Engine
     config.angular_templates.htmlcompressor = false
 
     config.before_configuration do |app|
-      config.angular_templates.inside_paths = [Rails.root.join('app', 'assets')]
+      config.angular_templates.inside_paths = [File.join('app', 'assets')]
```

Comment:
Not sure it matters just pointing out File.join returns a different type (String rather than Pathname).